### PR TITLE
chore: Goodbyy ABManifest

### PR DESCRIPTION
**Pull request by dalkia**
# Pull Request Description

### Prerequisites
- [ ] List any required setup steps
- [ ] Include environment/configuration requirements

### Test Steps
1. First step
2. Second step
3. Expected result after step 2
4. ...

### Additional Testing Notes
- Note any edge cases to verify
- Mention specific areas that need careful testing
- List known limitations or potential issues

## Quality Checklist
- [ ] Changes have been tested locally
- [ ] Documentation has been updated (if required)
- [ ] Performance impact has been considered
- [ ] For SDK features: Test scene is included

## Code Review Reference
Please review our [Code Review Standards](https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md) before submitting.

### CODE REVIEW

File: Explorer/Assets/DCL/AvatarRendering/Loading/Components/IAvatarAttachment.cs [hunk 1]
```diff
@@ -149,15 +145,13 @@ public static bool TryGetContentHashByKey(this IAvatarAttachment avatarAttachmen
             return false;
         }
 
-        public static void UpdateManifest(this IAvatarAttachment avatarAttachment, StreamableLoadingResult<SceneAssetBundleManifest> result)
+        public static void UpdateManifest(this IAvatarAttachment avatarAttachment)
```
[the remark this answers is not in the document]
Comment (dalkia) [reply Explorer/Assets/DCL/AvatarRendering/Loading/Components/IAvatarAttachment.cs:148]: Done

File: Explorer/Assets/DCL/Ipfs/AssetBundleRegistryEnum.cs [hunk 2]
```diff
@@ -1,9 +1,101 @@
-﻿namespace DCL.Ipfs
+﻿using DCL.Platforms;
+using System;
+
+namespace DCL.Ipfs
 {
     public enum AssetBundleRegistryEnum
```
[the remark this answers is not in the document]
Comment (dalkia) [reply Explorer/Assets/DCL/Ipfs/AssetBundleRegistryEnum.cs:6]: This enum was previously used in the status windows to show the state of the Asset Bundle conversion of the scene, and it was incorrectly deleted. It used to have this form:

image

Creating a [ticket](https://github.com/decentraland/unity-explorer/issues/5271) to track it and see if we want to bring it back

File: Explorer/Assets/DCL/Infrastructure/ECS/StreamableLoading/AssetBundles/LoadAssetBundleManifestSystem.cs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            urlBuilder!.Clear();
+
+            urlBuilder.AppendDomain(assetBundleURL)
+                      .AppendSubDirectory(URLSubdirectory.FromString("manifest"))
+                      .AppendPath(URLPath.FromString($"{hash}{PlatformUtils.GetCurrentPlatform()}.json"));
+
+            SceneAbDto sceneAbDto = await webRequestController.GetAsync(new CommonArguments(urlBuilder.Build(), RetryPolicy.WithRetries(1)), ct, reportCategory)
+                                                              .CreateFromJson<SceneAbDto>(WRJsonParser.Newtonsoft, WRThreadFlags.SwitchBackToMainThread);
+
+            AssetValidation.ValidateSceneAbDto(sceneAbDto.Version, hash);
```
[the remark this answers is not in the document]
Comment (dalkia) [reply Explorer/Assets/DCL/Infrastructure/ECS/StreamableLoading/AssetBundles/LoadAssetBundleManifestSystem.cs:60]: Im chabing it to a Error Log. In theory, we have reconverted everything; I think that just leaving a error log for later debugging is enough. No need to assert
